hearpreprocess/pipeline.py

```diff
@@ -798,6 +867,19 @@ def run(self):
                 indent=True,
             )
 
+            # Save the slug and the label in as the split metadata
+            audiolabel_df.to_csv(
+                self.workdir.joinpath(f"{split_path.stem}.csv"),
```

Comment:
Please don't use split_path.stem three times. Just once is fine, but if you're going to reuse it, give it a proper name (`split`) and make sure its in SPLITS